docs/embedding.md

```diff
@@ -71,131 +56,108 @@ You can pass [Webchat Options](#webchat-options) as an additional argument to th
+| teaserMessage | object | see [Teaser Message](#teaser-message) | Configure the Teaser Message, shown to users while the Webchat is still minimized |
+| chatOptions | object | see [Chat Options](#chat-options) | Configure the Chat Options, providing further options to your users |
+| privacyNotice | object | see [Privacy Notice](#privacy-notice) | Configure the privacy notice, optionally shown to be accepted by users before any message is sent  |
+| fileAttachmentMaxSize | number | 10485760 | The max size for file attachments uploaded by users **in bytes**, set to **10MB** as default |
+| maintenance | object | see [Maintenance](#maintenance)  | Configures the maintenance mode to prevent the user from using the Webchat during maintenance |
+| demoWebchat | object | see [Demo Webchat](#demo-webchat) | Configure the Cognigy.AI demo webchat. Only relevant if you copy the Demo Webchat implementation. |
+| embeddingConfiguration | object | see [Embedding Configuration](#embedding-configuration) | Settings related to the Webchat browser embedding. **Not configurable via Endpoint Editor** |
+| widgetSettings | object | see [Widget Settings](#widget-settings) | Additional Settings to configure the webchat widget behavior. **Not configurable via Endpoint Editor** |
 
 See it in action:  
-[![Edit Override Endpoint Settings](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/embedding-the-cognigy-webchat-bpz1r?fontsize=14&hidenavigation=1&theme=dark)
-
-#### Unread Messages Preview
+[![Edit Override Endpoint Settings](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/override-endpoint-settings-5gqm6g?file=%2Findex.html)
+
+#### Layout
+| Name | Type | Default | Description |
+| - | - | - | - |
+| title | string | `""` | The text that will be shown as header title of the Webchat |
+| logoUrl | string | `""` | The Logo shown in the header of the Webchat |
+| useOtherAgentLogo | boolean | `false` | Enables setting specific names and logos for bot and agent avatars. By default title and logoUrl will be used for the avatars. |
+| botAvatarName | string | `""` | Name for the AI Agent avatar which will be displayed above each AI Agent message. |
+| botLogoUrl | string | `""` | AI Agent avatar which will be displayed above each AI Agent message. Recommended img size: 28px x 28px |
+| agentAvatarName | string | `""` | Name for the Human Agent avatar which will be displayed above each Human Agent message |
+| agentLogoUrl | string | `""` | Human Agent avatar which will be displayed above each Human Agent message. Recommended img size: 28px x 28px |
+| inputAutogrowMaxRows | number | `4` | Maximum Number of Input Rows  |
+| enableInputCollation | boolean | `true` | If enabled, messages will be combined into a single message, dependent on the time set with `inputCollationTimeout` |
```

Comment:
default Value should be false. See [spec ](https://cognigy.visualstudio.com/Engineering/_git/technical-design-documents/pullrequest/24778?_a=files&path=/docs/Products/Webchat/concepts/webchat-v3-endpoint/index.md). I have fixed this in my [tests PR](https://github.com/Cognigy/WebchatWidget/pull/380/files#diff-278c6a88ca5523f8d2b36db48728f8a3c4129c3ef2af21d21d31d5492b9b8a28)

Reply:
fixed